syntax = "proto3";

package test.envoy.config.xds;

// Configuration for TestXdsResourcesDelegate.
message TestXdsResourcesDelegateConfig {
}
